openAi.py

```diff
@@ -17,28 +20,35 @@ async def get_chat_response(messages):
     )
     return response.choices[0].message["content"]
 
-async def main():
-    # Initialize a conversation with a system message
-    conversation = [{"role": "system", "content": "You are a dentist who gives super short answers to your patients and wants to know about all the symptoms they have."}]
+def get_user_input():
+    return input("User: ")
+
```

Comment:
U can call it name like telegram_input_MOCK
that wll explain that is only a mock for run the program on your pc .